### ISSUE TITLE
Balloon crashes ashita addons plugin on "..." or "......" during cutscenes

### ISSUE THREAD

**Issue by xythrez** [onimitch/ffxi-balloon-ashitav4#1]
Ashita API Ver: 4.16 (Git commit 4f0c5f20d7bf270608c529af74c48e85b3580bfb)
Balloon ver: 4.0.2

Other loaded addons (shouldn't matter, confirmed crash is reproducible with only balloon loaded):
- fps
- move
- drawdistance
- allmaps
- tcrossbar
- hxui
- zonename
- nomount

Steps to reproduce:
- Enter any story cutscene where a character says "..." or "......".
- The cutscene dialogue must consist of only dots. Cutscenes with "Speaker : ...... blah blah" or "Speaker : ...!?" are fine.

Expected result:
- Addon should show a text box with the said text

Actual result:
- AshitaV4 addons plugin immediately crashes, causing it to unload all addons

Logs listed below:
![Capture2](https://github.com/onimitch/ffxi-balloon-ashitav4/assets/46275090/3053f9fe-cc23-4312-a978-f0f1b047f53f)

![Capture](https://github.com/onimitch/ffxi-balloon-ashitav4/assets/46275090/da0a36ff-819a-4c80-8a7f-a7766879817d)

![Capture](https://github.com/onimitch/ffxi-balloon-ashitav4/assets/46275090/726ed04d-bcc7-4510-a97d-9a407400b2c3)

settings.lua:
```
require('common');

local settings = T{ };
settings["position"] = T{ };
settings["theme"] = "default";
settings["scale"] = 2;
settings["position"]["y"] = 1918;
settings["position"]["x"] = 1924;
settings["portraits"] = true;
settings["no_prompt_close_delay"] = 10;
settings["display_mode"] = 2;
settings["text_speed"] = 100;
settings["move_close"] = true;

return settings;
```

**Comment by onimitch** [comment]
Thanks for the detailed bug report, it's really appreciated!
I have some ideas about what might be the cause but I can't look into it until tomorrow. I'll get a fix out asap.

**Comment by onimitch** [comment]
Hello! Sorry for the delay. I've now fixed this and new version 4.0.3 is here: https://github.com/onimitch/ffxi-balloon-ashitav4/releases/tag/4.0.3

Thanks again for reporting the bug!
